addons/website/models/res_users.py

```diff
@@ -26,6 +25,20 @@ def _has_unsplash_key_rights(self):
             return True
         return super(ResUsers, self)._has_unsplash_key_rights()
 
+    @api.constrains('login', 'website_id')
+    def _check_login(self):
+        """ Do not allow two users with the same login without website """
+        self.flush(['login', 'website_id'])
+        self.env.cr.execute(""" SELECT login
+            FROM res_users
+            WHERE login IN (SELECT login FROM res_users WHERE id IN %s AND website_id IS NULL)
+                AND website_id IS NULL
+            GROUP BY login
+            HAVING COUNT(*) > 1
+        """, (tuple(self.ids),))
+        if self.env.cr.rowcount:
+            raise ValidationError(_('You can not have two users with the same login!'))
```

Comment:
```you cannot``` or ```you can't```

Reply:
> `you cannot` or `you can't`

Even if you're right, I prefer to keep the same error message as the sql_constraint (which is already translated).